beacon-chain/sync/deadlines.go

```diff
@@ -7,7 +7,18 @@ import (
 	"github.com/prysmaticlabs/prysm/shared/roughtime"
 )
 
+const defaultReadDuration time.Duration = 5
+const defaultWriteDuration time.Duration = 10
+
 func setRPCStreamDeadlines(stream network.Stream) {
-	stream.SetReadDeadline(roughtime.Now().Add(5 * time.Second))   // TTFB_TIMEOUT
-	stream.SetWriteDeadline(roughtime.Now().Add(10 * time.Second)) // RESP_TIMEOUT
+	setStreamReadDeadline(stream, defaultReadDuration)
+	setStreamWriteDeadline(stream, defaultWriteDuration)
+}
+
+func setStreamReadDeadline(stream network.Stream, duration time.Duration) {
+	stream.SetReadDeadline(roughtime.Now().Add(duration * time.Second)) // TTFB_TIMEOUT
```

Comment:
It doesn't make sense to pass a duration then multiply it by seconds. Either accept a uint64 for "deadlineInSeconds" or a duration as is. I prefer the latter option. 

```suggestion
	stream.SetReadDeadline(roughtime.Now().Add(duration))
```